grafana/datasources.libsonnet

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Can we put this file in a separate directory? `setup/`  or something?

Reply:
I half see where you're going, but am not really sure. `objects/` perhaps makes a bit of sense. Or move them to grafonnet.